### DISCUSSION TITLE
I can't create a project by "serverless create --template aws-nodejs --path myService"

### DISCUSSION THREAD

**Discussion by zl-fire** [serverless/serverless#10807] [Q&A] [answered]
**This is how I installed serverless**

<img width="1725" alt="image" src="https://user-images.githubusercontent.com/25565146/156924278-57e82d66-9cb8-49bd-9aa5-e22f7b058f82.png">

**Then I always get an error when I try to create a project**

<img width="679" alt="image" src="https://user-images.githubusercontent.com/25565146/156924330-0c0d4f7e-6e5c-4128-bad5-dfc0d8842ec1.png">

**Who can tell me why? Thank you.**

**Comment by mnapoli** [comment, marked as answer]
Hi!

Users in China are presented with a setup centered around the Chinese [Tencent](https://intl.cloud.tencent.com/) provider. If you are based in China and prefer to use AWS, set the following environment variable: `SERVERLESS_PLATFORM_VENDOR=aws`.

**Reply by tistisman** [reply]
I am not in china, we are very far from it.

also env variable doesn't work in windows.
